### ISSUE TITLE
Cannot build Android on fresh React Native 0.62.2 project

### ISSUE THREAD

**Issue by salockhart** [unimodules/react-native-unimodules#137]
### Steps to Reproduce:

1. Create a new React Native project running v0.62.2:
`npx react-native init UnimodulesBuildFailure`

2. Install `react-native-unimodules` and follow the setup guide for Android in the README

### Result

Running `npm run android` shows the following error:

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':expo-image-loader'.
> Failed to notify project evaluation listener.
   > wrong number of arguments
   > KotlinJvmAndroidCompilation with name 'debugAndroidTest' not found.
```

Here is a demo app: https://github.com/salockhart/UnimodulesBuildFailure

**Comment by salockhart** [comment]
Might be related to https://github.com/gradle/gradle/issues/11084

**Comment by salockhart** [comment]
This appears to be resolved if I update to `gradle 6.2` and `com.android.tools.build:gradle:3.4.2`
